server/service/src/vaccination/generate.rs

```diff
@@ -0,0 +1,71 @@
+use repository::StockLine;
+use util::uuid::uuid;
+
+use crate::{
+    invoice::{InsertPrescription, UpdatePrescription, UpdatePrescriptionStatus},
+    invoice_line::stock_out_line::{InsertStockOutLine, StockOutType},
+};
+
+#[derive(Debug)]
+pub struct CreatePrescription {
+    pub insert_prescription_input: InsertPrescription,
+    pub insert_stock_out_line_input: InsertStockOutLine,
+    pub update_prescription_input: UpdatePrescription,
+}
+
+pub fn generate_create_prescription(
```

Comment:
Nothing actually new in here, just pulled out the create prescription from the insert method, to share with update